lib/mayaUsd/ufe/UsdBatchOpsHandler.cpp

```diff
@@ -20,10 +20,7 @@
 namespace MAYAUSD_NS_DEF {
 namespace ufe {
 
-UsdBatchOpsHandler::UsdBatchOpsHandler()
-    : Ufe::BatchOpsHandler()
-{
```

Comment:
The base clas vary depending on the version of UFE, so dont call it explicitly. It's not needed anyway.